modules/customizing-the-home-page/proc-customizing-the-home-page-cards.adoc

```diff
@@ -56,6 +64,7 @@ dynamicPlugins:
 
 Search::
 +
+You can use the *SearchBar* card to provide essential search functionality directly on the Home page. 
 --
```

Comment:
Consider fixing block indentation on the entire module with list continuation markup. See https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#list-continuation

```suggestion
+
--
```